drfpasswordless/__version__.py

```diff
@@ -0,0 +1,3 @@
+VERSION = (1, 1, 5)
```

Comment:
let's change this to (1, 2, 0) and update it to 1.2.0 in __init__.py so I can upload to Pypi

Reply:
for sure, I didn't want to change it since I can't upload!

Reply:
@djstein yeah, I can take care of this later myself once we get tests to pass too